crates/asm/txs/bridge-v1/src/deposit_request/aux.rs

```diff
@@ -29,8 +53,12 @@ impl DrtHeaderAux {
     }
 
     /// Returns the execution environment address
-    pub fn ee_address(&self) -> &[u8; 20] {
-        &self.ee_address
+    pub fn dest_acct_serial(&self) -> &AccountSerial {
+        &self.dest_acct_serial
+    }
+
+    pub fn dest_subject(&self) -> &SubjectBytes {
+        &self.dest_subject
```

Comment:
I think these functions can be `const` no?

Reply:
There is a `clippy` lint (`clippy.missing_const_for_fn`) and was initially proposed in  https://github.com/alpenlabs/alpen/pull/836 but overruled by @delbonis (reason in https://github.com/alpenlabs/alpen/pull/836#pullrequestreview-2882391202).

Hence, I do keep in mind that some functions can (and should be) `const` in reviews.